com.unity.render-pipelines.core/Runtime/Volume/Volume.cs

```diff
@@ -19,7 +19,7 @@ public class Volume : MonoBehaviour
         /// <summary>
         /// The Volume priority in the stack. A higher value means higher priority. This supports negative values.
         /// </summary>
-        [Tooltip("Sets the Volume priority in the stack. A higher value means higher priority. You can use negative values.")]
+        [Tooltip("A value which determines which Volume it uses when Volumes have an equal amount of influence on the Scene. Volumes with a higher priority are rendered first.")]
```

Comment:
I think this tooltip is a bit misleading. 
I propose: "A value that determines precedence when two affecting volumes are overriding the same properties. Higher values take precedence over lower values."

Reply:
@TheoWong-pixel @oleks-k your thoughts?

Reply:
Yeah I'm down for this change :)

Reply:
Changed!